version/wintrust/certificate_windows.go

```diff
@@ -48,7 +48,9 @@ func ExtractCertificates(path string) ([]x509.Certificate, error) {
 			break
 		}
 		buf := make([]byte, cert.Length)
-		copy(buf, (*[1 << 20]byte)(unsafe.Pointer(cert.EncodedCert))[:])
+		for i := range buf {
+			buf[i] = *(*byte)(unsafe.Pointer(uintptr(unsafe.Pointer(cert.EncodedCert)) + uintptr(i)))
```

Comment:
Oooof... That's ugly. But I guess we have to wait until `unsafe.Slice` is added before we can make this prettier, right?

Reply:
Yeaaaaah. I see nothing prettier we could use today.

Reply:
I could add least add a copy of the unsafe add pointer wrapper that's coming in Go 1.16?

Reply:
Done.

Reply:
Oh, sure. That latter option seems better than my TODO comment.

Reply:
Oh, I see what you did in the new commit -- `unsafe.Add`. I was thinking something along the lines of the new `unsafe.Slice`, which could then just be passed to `copy`.

Reply:
Yeah. But `func add` can be added today. (and `unsafe.Add` might come in Go 1.16, while `unsafe.Slice` might be Go 1.17?). Either way. The code you merged with the TODO is also fine.

Reply:
Ahh, well. I think I'll leave it be. And then when 1.16 comes, it'll be another big global cleanup operation to make all the pointer arithmetic less nasty using `.Add`. And then for 1.17, there'll be another series of cleanups...